### PR TITLE
Loading Spinner!

### PR DESCRIPTION
**Pull request by egiurleo**
Our site doesn't currently have a loading state! I implemented some little loading dots that will appear on the jobs and meetups pages while the content loads. Not super visible with a fast internet connection, but it should make the experience of using our site a lot better for those with slow connections.

https://user-images.githubusercontent.com/9601737/151705438-ba4e894f-e576-41ef-82d3-73983eadddf5.mov

### CODE REVIEW

File: app/javascript/components/LoadingSpinner.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import 'stylesheets/loading-spinner';
+
+const LoadingSpinner = ({ className }) => (
```
Comment (egiurleo) [inline app/javascript/components/LoadingSpinner.jsx:6]: I know it technically isn't a "spinner," so... I'm open to better name suggestions?

File: app/javascript/components/pages/Jobs.jsx [hunk 2]
```diff
@@ -14,6 +15,8 @@ import 'stylesheets/page';
 import 'stylesheets/jobs';
 
 const Jobs = () => {
+    // eslint-disable-next-line no-unused-vars
+    const [loading, setLoading] = useState(true);
```
Comment (egiurleo) [inline app/javascript/components/pages/Jobs.jsx:19]: This sets the `loading` state to true until the jobs are fetched from the backend, at which point loading becomes false and the jobs are displayed.

Comment (stephanieminn) [reply app/javascript/components/pages/Jobs.jsx:19]: is the linter still complaining even though `loading` is called on L56? 🤔

File: app/javascript/components/LoadingSpinner.jsx (hunk 1 above)
Comment (stephanieminn) [reply app/javascript/components/LoadingSpinner.jsx:6]: `LoadingStatus`? haha idk I'm fine with it either way

**Review by stephanieminn — APPROVED** [review]

File: app/javascript/components/pages/Jobs.jsx (hunk 2 above)
Comment (egiurleo) [reply app/javascript/components/pages/Jobs.jsx:18]: Nope I deleted this line and never committed :) Thanks for catching!

File: app/javascript/components/LoadingSpinner.jsx (hunk 1 above)
Comment (egiurleo) [reply app/javascript/components/LoadingSpinner.jsx:6]: I'll leave it for now and if someone wants to fight me they totally can

**Pull request merged**